types/jquery/index.d.ts

```diff
@@ -680,6 +680,14 @@ interface JQueryCoordinates {
 }
 
 /**
+ * The interface used to specify the properties parameter in css()
+ */
+type cssPropertySetter = (index: number, value: string) => string | number;
```

Comment:
It looks like this `css` overload is just the overload above it applied multiple times. Jquery will always provide a `value` parameter, it is optional whether you want to use it or not.